website/docs/components/application-state/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
```

Comment:
For the overview, I am conflicted about how to add more detail without re-stating what's in the documentation. So I'm wondering if it would be wise to;
• Re-use the page cover description again?
• Or do you think I should add more detail to it?
---
Use the ApplicationState to provide information about the current state of the application.